src/matUtils/convert.cpp

```diff
@@ -478,7 +478,7 @@ void write_json_from_mat(MAT::Tree* T, std::string output_filename, std::vector<
         {
             "meta", {
                 {"title",title},
-                {"filters",json::array({"country"})},
+                {"filters",json::array({"country","userOrOld"})},
```

Comment:
I'm confused why `country` filtering wasn't already possible.

For some reason, the filters array in Usher auspice.json was empty. Have you checked that it now contains `"country"`? @jmcbroome